### DISCUSSION TITLE
Process to add new module (Apache Pinot Support)

### DISCUSSION THREAD

**Discussion by abhioncbr** [testcontainers/testcontainers-java#9175] [Q&A] [answered]
Hi,

I want some help adding the new module to test containers. I am a contributor to [Apache Pinot](https://pinot.apache.org/) and am wondering what should be done to add the support of Pinot in the test container. I would appreciate any reference or document. Thanks

**Comment by eddumelendez** [comment, marked as answer]
Hi @abhioncbr, thanks for the interest on contributing to the project with a module for Apache Pinot. Your contribution will be very welcome. Besides adding a new module under `modules`, please take a look at the [PR template](https://raw.githubusercontent.com/testcontainers/testcontainers-java/main/.github/pull_request_template.md)
